README.rst

```diff
@@ -4,6 +4,15 @@ FogLAMP "email notification" plugin
 
 Email notification plugin
 
+Note:
+
+This plugin requires the installation of libcurl-dev apt package and that
+is a virtual package provided by 'libcurl4-openssl-dev' among other options.
+
+.. code-block:: console
+
+  $ sudo apt-get install libcurl4-openssl-dev
```

Comment:
Got this error on Rpi:
`Err:1 http://raspbian.raspberrypi.org/raspbian stretch/main armhf libcurl4-openssl-dev armhf 7.52.1-5+deb9u8
  404  Not Found [IP: 5.153.225.207 80]
E: Failed to fetch http://raspbian.raspberrypi.org/raspbian/pool/main/c/curl/libcurl4-openssl-dev_7.52.1-5+deb9u8_armhf.deb  404  Not Found [IP: 5.153.225.207 80]
E: Unable to fetch some archives, maybe run apt-get update or try with --fix-missing?`

Reply:
I think other people outside are also getting this issue:

https://www.raspberrypi.org/forums/viewtopic.php?t=232928&p=1425878

Reply:
Yeah, that should be the reason. Approving for now as it works fine on ubuntu 16 and 18. Will verify it when the issue is resolved on Rpi.